lms/envs/certificates_html_view.json

```diff
@@ -0,0 +1,38 @@
+{
+    "default": {
+        "accomplishment_class_append": "accomplishment--certificate--honorcode",
```

Comment:
I've pulled all of the static strings out of this JSON file so the only things remaining are values which should not need to be translated